pkg/util/utils.go

```diff
@@ -156,6 +158,28 @@ func CalculateSubnetSize(mask int) int64 {
 }
 
 func IsSystemNamespace(c client.Client, ns string, obj *v1.Namespace) (bool, error) {
+	// first check t1 then vpc
+	isSysNs, err := IsT1Namespace(c, ns, obj)
```

Comment:
It looks like this part of logic will also be called when VPC model is enabled. We might need to make two branches to handle T1 and VPC mode.